docs/V1/SPEC.md

```diff
@@ -0,0 +1,204 @@
+# Nation3 Court Jurisdiction V1 <!-- omit in toc -->
```

Comment:
Is `docs/V1/SPEC.md` a duplicate of `docs/SPEC.md`? Do we need to keep two versions of the same document?

Reply:
I'm thinking of having the general spec of the system independent from the version and then a more detailed spec on how each version is implemented.